src/components/ItemList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import store from '../store/store.jsx';
+
+const ItemList = React.createClass({
+  render: function() {
+	  	return (
+	  		<div>
+	        <h1>Your Items:</h1>
+		        {store.getState().shirts.map(function(val, idx) {
```

Comment:
you could have add for (key and value loop ) through your store to save you from writing boilerplate code